.circleci/config.yml

```diff
@@ -34,12 +34,9 @@ jobs:
             - "/usr/local/bin"
             - "/usr/local/lib/python3.7/site-packages"
           key: deps-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
-      - run:
```

Comment:
https://pytest-django.readthedocs.io/en/latest/database.html#django-db-setup

"The default implementation creates the test database by applying migrations and removes databases after the test run."

So we shouldn't need this extra step here